examples_tests/common/QToQuitEventReceiver.h

```diff
@@ -5,31 +5,31 @@
 #ifndef __NBL_Q_TO_QUIT_EVENT_RECEIVER_H__INCLUDED__
 #define __NBL_Q_TO_QUIT_EVENT_RECEIVER_H__INCLUDED__
 
-#include "nabla.h"
+#include <nabla.h>
 
-//! Simple event receiver for most examples that closes the engine when Q is pressed
-class QToQuitEventReceiver : public nbl::IEventReceiver
+using namespace nbl;
+using namespace ui;
+
+/*
+	Simple event receiver for most examples
+	that closes the engine when Q is pressed
+*/
+
+class QToQuitEventReceiver
```

Comment:
get rid of this @Erfan-Ahmadi & @AnastaZIuk , it was always a placeholder...

Now you have a camera that doesn't hijack the screen, so just waiting on the window closed event callback is sufficient (because one can finally press the Close button)

Reply:
@AnastaZIuk I will take care of this issue in this branch. 
I will let you know when to merge it into your examples.

Reply:
@AnastaZIuk It's pushed now.
you should be able to get windowCb easily from CommonAPI `initOutput` and use it's `isWindowOpen` function
https://github.com/Devsh-Graphics-Programming/Nabla/pull/186/commits/1761843f6613615fc17850b09991b5a61f18e41f
If your examples include QToQuitEventReceiver, It's file is now removed.

Reply:
thank you @Erfan-Ahmadi, I will merge the branch then